docs/stable/2-account.md

```diff
@@ -213,12 +220,193 @@ All messages sent are encrypted with the public key of the destination and signe
 
 -->
 
+## Public Key Compression
+
+The node MUST provide functionality for the compression and decompression of public / chat keys.
+
+For maximum flexibility the node MUST support public keys encoded in a wide range of encoding formats, detailed below.
+
+For future and backward compatibility with Ethereum networks the node MUST supports at least 3 elliptical curve public key formats, detailed below.
+
+### Basic Compression Example
+
+In the example of a typical hexadecimal encoded elliptical curve (EC) public key (such as a secp256k1 pk),
+
+```text
+0x04261c55675e55ff25edb50b345cfb3a3f35f60712d251cbaaab97bd50054c6ebc3cd4e22200c68daf7493e1f8da6a190a68a671e2d3977809612424c7c3888bc6
+```
+
+minor modification for compatibility and flexibility makes the key self-identifiable and easily parsable, 
+
+```text
+fe70104261c55675e55ff25edb50b345cfb3a3f35f60712d251cbaaab97bd50054c6ebc3cd4e22200c68daf7493e1f8da6a190a68a671e2d3977809612424c7c3888bc6
+```
+
+EC compression and compact encoding produces a much smaller string representation of the original key.
+
+```text
+zQ3shPyZJnxZK4Bwyx9QsaksNKDYTPmpwPvGSjMYVHoXHeEgB
+```
+
+### Key Encoding
+
+The node MUST use the [multiformats/multibase](https://github.com/multiformats/multibase) encoding protocol to interpret incoming key data and to return key data in a desired encoding.
+
+The node MUST support the following `multibase` encoding formats.
```

Comment:
why is this a MUST? what happens if they don't?

Reply:
Hmm, I actually can't come up with a good justification for why this is a MUST. I will downgrade to a SHOULD.